app/utils/lava_log_parser.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Based on log2html.py:
+# https://git.linaro.org/people/kevin.hilman/build-scripts.git/blob_plain/HEAD:/log2html.py
```

Comment:
My git repos at Linaro are long dead.  The right URL for that mess, er, script is: 
https://github.com/khilman/lab/blob/master/log2html.py

Otherwise, really nice to see this going into the backend.  Thanks!